tests/unit/lang/translator/de.txt

```diff
@@ -0,0 +1,162 @@
+\nBootloader Version\n%d.%d.%d\n\nBuddy Board\n%d.%d.%d\n%s
+Ein Crash-Dump-Bericht (Datei dump.bin) wurde auf dem USB-Laufwerk gespeichert.
+Alle Tests erfolgreich abgeschlossen!
+Immer
+Anwenden
+Druck wirklich beenden?
+Assist
+Auto Home
+AutTn
+ZURÜCK
+BlindAlert
+KnopfEcho
```

Comment:
???

Reply:
These are dummy strings, will be removed in the next batch of translations.